matisse/src/main/java/com/zhihu/matisse/internal/ui/widget/CheckView.java

```diff
@@ -84,7 +85,10 @@ private void init(Context context) {
         mStrokePaint.setStyle(Paint.Style.STROKE);
         mStrokePaint.setXfermode(new PorterDuffXfermode(PorterDuff.Mode.SRC_OVER));
         mStrokePaint.setStrokeWidth(STROKE_WIDTH * mDensity);
-        mStrokePaint.setColor(Color.WHITE);
+        TypedArray ta = getContext().getTheme().obtainStyledAttributes(new int[]{R.attr.item_checkCircle_borderColor});
+        int color = ta.getColor(0, 0);
```

Comment:
Better pass a default value which is `@color/zhihu_item_checkCircle_borderColor` cause someone may derive a custom theme from a system built-in theme.